JSTests/wasm/stress/fsub-nan-copysign.js

```diff
@@ -0,0 +1,21 @@
+import * as assert from '../assert.js';
+
+// (module
+//     (type (func (param f64) (result f64)))
+//     (func (type 0) (local f64)
+//         (local.set 1 (local.get 0))
+//         (local.set 0 (f64.sub (local.get 0) (f64.const nan)))
+//         (local.set 1 (f64.copysign (local.get 1) (local.get 0)))
+//         (local.get 1)
+//     )
+//   (export "poc" (func 0))
+// )
+
+const wasm_code = new Uint8Array([0, 97, 115, 109, 1, 0, 0, 0, 1, 6, 1, 96, 1, 124, 1, 124, 3, 2, 1, 0, 7, 7, 1, 3, 112, 111, 99, 0, 0, 10, 33, 1, 31, 1, 1, 124, 32, 0, 33, 1, 32, 0, 68, 0, 0, 0, 0, 0, 0, 248, 127, 161, 33, 0, 32, 1, 32, 0, 166, 33, 1, 32, 1, 11]);
```

Comment:
Can we get a test for the non-const and the const-const case too?

Reply:
Done